tests/regression/1_run_regression_test.sh

```diff
@@ -0,0 +1,18 @@
+git fetch origin develop
```

Comment:
Insert `#!/usr/bin/env bash` in the first line to indicate the appropriate interpreter.
Reference: https://github.com/jserv/MazuCC/blob/master/tests/driver.sh